canonical/maas-ui
# refactor: MachineListTable

## Done

- refactor: MachineListTable
  - extract tableModels to own file

## QA

### QA steps

- Go to machine list
- Verify the machine list loaded and pagination, grouping and row selection works as before.
